docs/src/language_servers.md

```diff
@@ -540,6 +540,26 @@ Follow installation instructions on [LSP-typescript](https://github.com/sublimel
 
 Follow installation instructions on [LSP-vue](https://github.com/sublimelsp/LSP-vue).
 
+## Vala
+1. Install [Vala-TMBundle](https://github.com/technosophos/Vala-TMBundle) package to add Vala syntax highlighting and for Vala files to be reconginsed.
+2. Install the [Vala Language Server](https://github.com/Prince781/vala-language-server)
```

Comment:
It looks like the syntax is on Package Control - https://packagecontrol.io/packages/Vala-TMBundle

It would be better to link to that or even directly say which package to install from Package Control